Client/Frontend/EnhancedTrackingProtection/ETPCoverSheetViewController.swift

```diff
@@ -33,72 +32,66 @@ class ETPCoverSheetViewController: UIViewController {
     struct UX {
         static let doneButtonPadding: CGFloat = 20
         static let doneButtonHeight: CGFloat = 20
-        static let primaryButtonColour = UIColor.Photon.Blue50
         static let primaryButtonCornerRadius: CGFloat = 10
-        static let primaryButtonFont = UIFont.systemFont(ofSize: 18, weight: .semibold)
         static let primaryButtonHeight: CGFloat = 46
         static let primaryButtonEdgeInset: CGFloat = 18
+        static let primaryButtonFontSize: CGFloat = 18
+        static let horizontalMargin: CGFloat = 16
+        static let largeVerticalMargin: CGFloat = 48
+        static let imageHeight: CGFloat = 260
     }
 
     // Public constants
     let viewModel = ETPViewModel()
-    static let theme = BuiltinThemeName(rawValue: LegacyThemeManager.instance.current.name) ?? .normal
-    // Private vars
-    private var fxTextThemeColour: UIColor {
-        // For dark theme we want to show light colours and for light we want to show dark colours
-        return ETPCoverSheetViewController.theme == .dark ? .white : .black
+    var notificationCenter: NotificationProtocol = NotificationCenter.default
+
+    private lazy var doneButton: UIButton = .build { button in
+        button.setTitle(.SettingsSearchDoneButton, for: .normal)
+        button.titleLabel?.font = UIFont.systemFont(ofSize: UX.primaryButtonFontSize,
+                                                    weight: .regular)
+        button.addTarget(self, action: #selector(self.dismissAnimated), for: .touchUpInside)
     }
-    private var fxBackgroundThemeColour: UIColor {
-        return ETPCoverSheetViewController.theme == .dark ? .black : .white
+
+    let pairImageView: UIImageView = .build { imageView in
```

Comment:
nit: we should make this private just like other views, unless this is used somewhere? 🤔